src/pages/watch-later/watch-later.js

```diff
@@ -1,11 +1,26 @@
 import styles from "./watch-later.module.scss"
 import { VideoCard, SidePane } from "@components"
-
+import { useWatchLater } from "@providers/watch-later-provider"
 const WatchLater = () => {
+    const { WatchLaterState } = useWatchLater()
     return (
         <div className={styles.container}>
             <SidePane title="Watch Later..." time="30 Minutes" count={30} />
             <div className={styles.main}>
+                {
+                    WatchLaterState.length !== 0
```

Comment:
suggestion - optional chaining can be used here for `watchLaterState.map()`. Please correct me if I'm wrong.

Reply:
Not able to understand what you are suggesting here

Reply:
I was suggesting that `watchlaterState?.map()`would also work if the state is empty. So you don't have to check for length seperately.

Reply:
Yeah that would work but i need else condition for empty list UI element